src/main/java/se/bjurr/prnfb/service/ButtonsService.java

```diff
@@ -126,19 +127,25 @@ private boolean isVisibleOnRepository(PrnfbButton button, Repository repository)
  }
 
  @VisibleForTesting
- void doHandlePressed(UUID buttonUuid, ClientKeyStore clientKeyStore, boolean shouldAcceptAnyCertificate,
+ Map<String, HttpResponse> doHandlePressed(UUID buttonUuid, ClientKeyStore clientKeyStore, boolean shouldAcceptAnyCertificate,
```

Comment:
Not sure here... the names are not unique. And defaults to just "Notification". Perhaps use the UUID? And frontend can lookup that notification using the UUID to find whatever information is needed.